packages/parrot-middleware/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "clean": "rimraf lib",
     "prebuild": "npm run clean",
```

Comment:
doesn't rollup require 
```json
 "main": "lib/cjs/index.js",
  "module": "lib/esm/index.js",
```
entrypoints?

Reply:
Not necessarily - it can get by with one entry point `src/index.js` and just output one format (`cjs`). If we want to add an `esm` exportable we can def do that as you suggested. I just want to ensure it returns to the way it was, no other factors, then we can add `esm`.

Reply:
So you think `esm` wasn't supported before? must be because before the library update where we dropped support for esm people could still use it in previous versions of this library

Reply:
It's not necessarily that we didn't support it, we could have but we were only bundling with `cjs` package format for `node` compatibility. I think in the future, we will have direct support for `module` in the package and bundle `esm` alongside `cjs`.